content/chronograf/v1.8/tools/chronograf-cli/_index.md

```diff
@@ -68,11 +68,12 @@ chronograf [flags]
-| `--auth-duration`      | Total duration, in hours, of cookie life for authentication. Default value is `720h`.                                                                                                                                                                                                               | `$AUTH_DURATION` |
-| `--public-url`         | Public URL required to access Chronograf using a web browser. For example, if you access Chronograf using the default URL, the public URL value would be `http://localhost:8888`. Required for Google OAuth 2.0 authentication. Used for Auth0 and some generic OAuth 2.0 authentication providers. | `$PUBLIC_URL`    |
+| Flag                                 | Description                                                                                                                                                                                                                                                                                         | Env. Variable    |
+|:-------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------|
+| `-t`, `--token-secret`               | Secret for signing tokens                                                                                                                                                                                                                                                                           | `$TOKEN_SECRET`  |
+| `--auth-duration`                    | Total duration, in hours, of cookie life for authentication. Default value is `720h`.                                                                                                                                                                                                               | `$AUTH_DURATION` |
+| `--public-url`                       | Public URL required to access Chronograf using a web browser. For example, if you access Chronograf using the default URL, the public URL value would be `http://localhost:8888`. Required for Google OAuth 2.0 authentication. Used for Auth0 and some generic OAuth 2.0 authentication providers. | `$PUBLIC_URL`    |
+| `—htpasswd <path to .htpasswd file>` | Password file for use with HTTP basic authentication (username and password).                                                                                                                                                                                                                       | `$HTPASSWD`      |
```

Comment:
Let's link to nginx doc here too & maybe describe what the htpasswd file is/whether it's installed with....nginx/Chronograf?